include/game.h

```diff
@@ -1564,6 +1595,55 @@ typedef struct {
     /* 0x40 */ u8* scriptEnd;    // pointer to the end of the script
 } Dialogue;                      // size = 0x44
 
+// st0_psp/3101C, st0/bss.c, st0/prologue_scroll.c, st0/3101C
+typedef struct {
```

Comment:
Ultimately, the set of Dialogue structs should probably be in cutscene.h. Currently Dialogue and DIalogue2 are used in st0/bss so collecting them here made more sense to me for now.